articles/active-directory/develop/msal-net-xamarin-android-considerations.md

```diff
@@ -71,26 +71,25 @@ protected override void OnActivityResult(int requestCode,
 }
 ```
 
-## Update the Android manifest
+## Update the Android Manifest for System Browser Support 
 
-The *AndroidManifest.xml* file should contain the following values:
+The *AndroidManifest.xml* file should contain the following values for system browser support:
```

Comment:
```suggestion
To support the system browser, the *AndroidManifest.xml* file should contain the following values:
```

Always present the goal first, then the how/what/why.

* Better scannability: Easier to spot relevant content when scanning instead of reading (which is nearly everyone).
* Simplification/concision: Having to read less to find out if it's a thing you need to pay attention to is always good. If I don't need to support the system the reader can bail halfway through the sentence. Like in code, you want to fail fast.